components/transactions/graphql/fragments.ts

```diff
@@ -4,7 +4,6 @@ import { accountHoverCardFields } from '../../AccountHoverCard';
 
 export const transactionsQueryCollectionFragment = gql`
   fragment TransactionsQueryCollectionFragment on TransactionCollection {
-    totalCount
```

Comment:
`totalCount` is nor part of this fragment anymore and will have to be explicitly asked when it's required.